Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:alpine as build
-RUN apk add make git gcc g++ 
+FROM registry.access.redhat.com/ubi8/ubi:8.2 as build
+RUN dnf install -y make git gcc gcc-c++ golang
```

Comment:
why is golang needed for the ubi image but not for the alpine image ?

Reply:
Because the alpine image indicated is actually the variant maintained by the golang team, which is Alpine _plus_ golang (note the full name `FROM golang:alpine`